### PR TITLE
Update Portuguese translation for 1.19.1

### PR DESCRIPTION
**Pull request by liddack**

### CODE REVIEW

File: resources/pt/cemu.po [hunk 1]
```diff
@@ -849,20 +831,21 @@ msgid ""
 "\n"
 "Recommended: Auto"
```
Comment (Fs00) [inline resources/pt/cemu.po:832]: 'Auto' should be 'Automático' since you've changed it.

File: resources/pt/cemu.po [hunk 2]
```diff
@@ -1872,11 +1913,11 @@ msgstr "&Separar tela do GamePad"
 
 #: MainWindow.cpp
 msgid "&General settings"
-msgstr "Configurações &gerais"
+msgstr "Configurações &gerais…"
 
 #: MainWindow.cpp
 msgid "&Input settings"
-msgstr "Configurações de &controles"
+msgstr "Configurações de &controles…"
```
Comment (Fs00) [inline resources/pt/cemu.po:1920]: Using ellipsis here clearly goes against the guidelines you linked:
> For example, the commands About, Advanced, Help, Options, Properties, and **Settings** must display another window when clicked, but don't require additional information from the user. Therefore they don't need ellipses.

File: resources/pt/cemu.po [hunk 3]
```diff
@@ -2048,19 +2093,19 @@ msgstr "&Depuração"
 
 #: MainWindow.cpp
 msgid "&Check for updates"
-msgstr "&Verificar atualizações..."
+msgstr "&Verificar atualizações"
 
 #: MainWindow.cpp
 msgid "&Getting started"
-msgstr "&Guia de início"
+msgstr "&Guia de início…"
```
Comment (Fs00) [inline resources/pt/cemu.po:2100]: I don't think ellipsis should go here too, according to the guidelines:
> This doesn't mean you should use an ellipsis whenever an action displays another window—**only when additional information is required** to perform the action.

File: resources/pt/cemu.po (hunk 1 above)
Comment (liddack) [reply resources/pt/cemu.po:828]: I think you mentioned the wrong line here, but this made me find another translation I forgot to fix. Thanks!

**Comment by liddack** [comment]
I added manual line breaks because the infotips on Cemu are displayed for a short period of time (~5 seconds), and some of them are too (horizontally) large to be completely read before disappearing, at the same time they would be faster to read if they were split in two or more lines.
So, to make up for the reduced exposure time, I followed some video captioning conventions to split phrases in a way that the reader could scan through them more easily.
But now that you mentioned these strings might be used in other ways, I understand it's better to remove manual line breaks for now.

File: resources/pt/cemu.po [hunk 4]
```diff
@@ -849,20 +827,21 @@ msgid ""
...
 msgstr ""
-"Shaders pré-compilados podem acelerar o tempo de carregamento dos shaders no "
-"início da emulação.\n"
-"A opção \"auto\" ativará isto para AMD/Intel, mas deixará desativado em GPUs "
-"da NVIDIA como solução para um bug no driver.\n"
+"Shaders pré-compilados podem acelerar o tempo de\n"
+"carregamento dos shaders no início da emulação.\n"
+"A opção \"Automático\" ativará isto para AMD/Intel,\n"
+"mas deixará desativado em GPUs da NVIDIA\n"
+"como solução para um bug no driver.\n"
```
Comment (Fs00) [inline resources/pt/cemu.po:834]: One last thing! You forgot to revert line breaks in this string. Everything else is fine!

**Comment by Fs00** [comment]
Thanks for your work!

Btw @TeamCemu it might be a good idea to increase the display time of long tooltips. Quoting @liddack from the message above:
> infotips on Cemu are displayed for a short period of time (~5 seconds), and some of them are too (horizontally) large to be completely read before disappearing

**Pull request merged**